### PR TITLE
new icon: thymeleaf (original, plain, original-wordmark, plain-wordmark)

### LINKED ISSUE #1657: [ICON REQUEST]: thymeleaf
### I have searched through the issues and didn't find my problem.

- [x] Confirm

### About the icon

Thyemeal is a Java template engine 

### Links and sources

[https://seeklogo.com/vector-logo/441827/thymeleaf](https://seeklogo.com/vector-logo/441827/thymeleaf)
[https://icon-sets.iconify.design/simple-icons/thymeleaf/](https://icon-sets.iconify.design/simple-icons/thymeleaf/)

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ruubytes**
## Double check these details before you open a PR

- [X] PR does not match another non-stale PR currently opened
- [X] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [X] PR's base is the `develop` branch.
- [X] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [X] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [X] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1657**

## Link to prove your SVG is correct and up-to-date.

<a href="https://www.thymeleaf.org/">

</a

### CODE REVIEW

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Looks good to me, but a plain or line icon is needed since `thymeleaf-original.svg` can not be used as a font, due to multiple paths and fill colors.

If you want to make a line icon, you can follow this guide: https://github.com/devicons/devicon/issues/1306#issuecomment-1297625294

Regardless if you choose plain or line, make sure it meets our [SVG Standards](https://github.com/devicons/devicon/wiki/SVG-Standards). 

If you want more info on the different SVG versions we use (e.g.) original, plain, line), you can read our [SVG Versions](https://github.com/devicons/devicon/wiki/SVG-Versions) wiki page.

**Comment by ruubytes** [comment]
aye aye captain

File: devicon.json [hunk 1]
```diff
@@ -9006,6 +9006,25 @@
...
+            "template",
+            "engine",
+            "server",
+            "side"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ]
```
Comment (Snailedlt) [inline devicon.json:9023]:
```suggestion
            ],
            "font": [
                "plain"
            ]
```

**Review by Snailedlt — DISMISSED** [review]
Wow, that was fast. Looks like "font" is the only thing missing

**Comment by ruubytes** [comment]
It didn't stop at the svgo-action, but the git auto commit action, what happened?

**Comment by Snailedlt** [comment]
@ruubytes Yeah, it's a known issue with the workflow -> #1715. Hopefully we can get around to fixing it soon, until the, you can safely ignore the Optimize Icons workflow entirely

**Review by lunatic-fox — APPROVED** [review]
Great work! ✔
Thank you for this contribution! 🚀

**Review by Panquesito7 — DISMISSED** [review]
Looks like there's a wordmark version for this logo (found it in their [downloads](https://www.thymeleaf.org/download.html) page). It'd be great if you could add those as well. 🙂
The other icons look great by the way. Thanks! 🚀

**Comment by Snailedlt** [comment]
Here's all the official svg logos: https://github.com/thymeleaf/thymeleaf-org/blob/main/artwork/thymeleaf%202016/thymeleaf.svg

**Review by lunatic-fox — APPROVED** [review]
I think that everything is ok right now. ✔️
Thank you by this contribution! 🚀
image
image

**Review by lunatic-fox — APPROVED** [review]

**Review by canaleal — APPROVED** [review]

**Pull request merged**